#it will throw an error
#name = 'zabeer'
#number = int(name)
#print(number)

try:
    name = 'zabeer'
    number = int(name)
    print(number)

except Exception as e:
    print(e)
